tutorials/lava/lib/peripherals/dvs/PropheseeCamera.ipynb

```diff
@@ -0,0 +1,313 @@
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 2,
+   "id": "602de9c0-4501-42b3-b604-589346b82b19",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "SEQUENCE_FILENAME_RAW = \"80_balls.raw\"\n",
+    "# if the file doesn't exist, it will be downloaded from Prophesee's public sample server\n",
+    "get_sample(SEQUENCE_FILENAME_RAW)\n",
+    "\n",
+    "reader = RawReader(SEQUENCE_FILENAME_RAW)\n",
+    "height, width = reader.get_size()\n",
+    "del reader"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "1e72821b-418a-451d-a52d-1e33f96faccb",
+   "metadata": {},
+   "source": [
+    "## Basic usage\n",
+    "\n",
+    "Initialize PropheseeCamera using the path to a recording file if RAW format, in order to use a live camera, replace the path with an empty string (\"\")."
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "f13c3774-0132-4858-b10d-94c0bac212c0",
+   "metadata": {},
+   "source": [
+    "\n",
+    "<table>\n",
+    "<tr>\n",
+    "    <td> <img src=\"gifs/basic.gif\" alt=\"Drawing\" style=\"height: 250px;\"/> </td>\n",
+    "</tr>\n",
+    "</table>"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 3,
+   "id": "2ce2d8f2-a440-4d4f-a86b-b0b62d3fbee9",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "\n",
+    "# Init Processes\n",
+    "frame_input = PropheseeCamera(device=SEQUENCE_FILENAME_RAW,\n",
+    "                              sensor_shape=(height, width),\n",
+    "                              num_output_time_bins=1)\n",
+    "\n",
+    "recv = VisProcess(shape=frame_input.s_out.shape)\n",
+    "\n",
+    "# Connect\n",
+    "frame_input.s_out.connect(recv.s_in)\n",
+    "\n",
+    "# Run\n",
+    "num_steps = 200\n",
+    "run_cfg = Loihi2SimCfg()\n",
+    "run_cnd = RunSteps(num_steps=num_steps)\n",
+    "\n",
+    "frame_input.run(condition=run_cnd, run_cfg=run_cfg)\n",
+    "frame_input.stop()"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "8a0a80a0-3d5f-4317-9a95-14b121bbf1d5",
+   "metadata": {},
+   "source": [
+    "## Apply filters\n",
+    "\n",
+    "As you see, the falling balls in the recording cause a trail of events. In many cases these trails are blurring the objects and are undesired. In order to reduce the trails, we can apply the TrailFilterAlgorithm from the metavision_sdk. See the metavision documentation for more filters and their usage."
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "6a9fc5fb-30d8-4cd4-bb0f-42b2fe0aa5ef",
+   "metadata": {},
+   "source": [
+    "\n",
+    "<table>\n",
+    "<tr>\n",
+    "    <td> <img src=\"gifs/filters.gif\" alt=\"Drawing\" style=\"height: 250px;\"/> </td>\n",
+    "</tr>\n",
+    "</table>"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 4,
+   "id": "73ef5962-56de-411c-ba02-2113163703d9",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "filters = [TrailFilterAlgorithm(width=width, height=height, threshold=100000),\n",
```

Comment:
Could you import the modules here only, rather than in the beginning of the tutorial? That may clarify that they come from metavision rather than from Lava.